src/main/java/com/kammatysiak/medicalclinic/exceptions/PatientPasswordValidationException.java

```diff
@@ -0,0 +1,9 @@
+package com.kammatysiak.medicalclinic.exceptions;
+
+import lombok.Getter;
+import org.springframework.http.HttpStatus;
+@Getter
+public class PatientPasswordValidationException extends RuntimeException {
```

Comment:
Zdefiniuj wyjątek MedicalClinicException, niech zawiera on pole HttpStatus status i rozszerza RuntimeException,
Natomiast kolejne wyjątki wszystkie definiowane przez nas niech rozszerzaja MedicalClinicException a nastepnie dostosuj handler oraz miejsca w ktorych rzucasz wyjatki tak aby to Service odpowiadał za to jaki komunikat zostanie zwrócony użytkownikowi.